python/cuml/cuml/tests/test_svm.py

```diff
@@ -334,7 +340,13 @@ def test_svc_weights(class_weight, sample_weight):
 
     sklSVC = svm.SVC(**params)
     sklSVC.fit(X, y, sample_weight)
-    compare_svm(cuSVC, sklSVC, X, y, coef_tol=1e-5, report_summary=True)
+    if not probability:
```

Comment:
Is there _something_ we can check (/instead) for `probability=True` ?

Reply:
We do check `score`, which hits all the internal attributes as they're needed for inference. FWIW this test as is caught some bugs, I don't view it as deficient to leave it in this state.

I'd rather not hack around this for now (as noted in the comment), and instead fix the internal representation. Was going to open a follow-up issue on that later today.

Reply:
Ok, good enough for me.